charts/timed/templates/cronjob-backend.yaml

```diff
@@ -32,7 +32,7 @@ spec:
                   valueFrom:
                     secretKeyRef:
                       name: {{ include "timed.postgresql.fullname" $ }}
-                      key: postgresql-password
+                      key: postgres-password
```

Comment:
isn't this the Postgres Admin PW as default?

https://github.com/bitnami/charts/blob/b1d2449d52bab5b498ea3b56837c6ecd63274fab/bitnami/postgresql/values.yaml#L149

Reply:
yes, but it's only used on ci

Reply:
it's in the `templates` so no, it's not only used on CI. If we would use the chart with the postgres dep, it would reference the Admin PW by default. 

I think this should be `password` instead of `postgres-password`

Reply:
>  If we would use the chart with the postgres dep

This PR is about not using the postgres dep, the only reason the dep is still here is for CI.

Reply:
This PR is about adding support for external postgresql Database. Not making the postgres dep always the admin password

Reply:
Just don't hardcode the usage of admin-password in the templates and I'm happy. If someone wants to deploy timed with this change and use the postgres dep, it would use the admin-password by default and that is not good. We can make it parameterized and add it to values and use ci/default-values.yaml to overwrite it to use the admin-password as key

Reply:
Think I changed it everywhere, but somewhere its expecting something different (in testing):
``` psycopg2.OperationalError: FATAL:  password authentication failed for user "postgres" ```

Switching back to WIP, until I figured this out.